cluster/src/main/java/com/hartwig/pipeline/report/PipelineResults.java

```diff
@@ -97,4 +97,12 @@ private void writeComplete(final String name) {
     private String path(final String name, final Folder folder, final String fileName) {
         return String.format("%s/%s%s", name, folder.name(), fileName);
     }
+
+    public void initialise(final Arguments arguments, final SingleSampleRunMetadata metadata) {
```

Comment:
Why is the STAGING_COMPETE flag written if the single sample pipeline failed? Is there any way we can fix this such that staging complete isn't written when CRAM fails and the somatic pipeline is never kicked off?